MesnardBarba-CiSE.md

```diff
@@ -128,7 +128,8 @@ We already know that boundary conditions at the outlet of the computational doma
 Our first attempt with IBAMR used a zero-gradient velocity boundary condition at the outlet. 
 This resulted in some blockage effect when the wake vortices reach the domain boundary: strong vorticity rebounds from the artificial boundary and propagates back to the domain (Figure 5). 
 Of course, this is unphysical and the result unacceptable. 
-After a long search in the literature and in the code documentation, we discovered that IBAMR needs us to select a "stabilized outlet," which is a boundary condition that acts like a force pushing the vortices out. 
+After exchanging with the main developers on the online forum , we found
```

Comment:
"After exchanging" what? This is not a correct expression. Also, I would leave the comment that we did a long search in the literature. So, maybe:
"After a long search in the literature and in the documentation, it was through a conversation with the main developers on the online forum that we discovered the solution: using a "stabilized outlet" boundary condition, which adds a forcing to push the vortices out.